src/bdk.udl

```diff
@@ -280,6 +280,8 @@ interface DescriptorSecretKey {
 
   DescriptorPublicKey as_public();
 
+  sequence<u8> secret_key_bytes();
```

Comment:
How about shortening the name to `secret_bytes()` same as the internal bdk function?

Reply:
Yeah let's do it. In this case I had made a deliberate choice because I actually don't like the API, (`DescriptorSecretKey.secret_bytes`? it's not explicit to me that it's about to return you the secret key). But I agree we should stick to the names we have in the Rust libs (and if I don't like it I should take it up with the upstream! haha).

Fixed.